data/mods/MindOverMatter/effectoncondition/eoc_drain_events.json

```
@@ -0,0 +1,189 @@
+          "math": [ "u_val('time: 30 s') * rng( ( u_val('vitamin', 'name:vitamin_psionic_drain') / 2 ), ( u_val('vitamin', 'name:vitamin_psionic_drain') * 2 ) )" ]
+        }
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_EXTRA_DRAIN",
+    "condition": { 
+      "and": [ 
+        { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", ">=", "15" ] },  
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "( (u_val('vitamin', 'name:vitamin_psionic_drain') * 2.5 ) + 20)"
+              ]
+            },
+            "y": 1000
+          }
+        } 
+      ] 
+    },
+    "effect": [
+      { "u_message": "You feel a sudden fatigue as your powers are unleashed.", "type": "bad" },
+      { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", "+=", "rng( 2,6 )" ] }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_NOSEBLEED",
+    "condition": { 
+      "and": [ 
+        { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", ">=", "25" ] },  
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "( (u_val('vitamin', 'name:vitamin_psionic_drain') * 1.8 ) + 40)"
+              ]
+            },
+            "y": 1000
+          }
+        } 
+      ] 
+    },
+    "effect": [
+      { "u_message": "As you unleash your powers, blood drips from your nose!", "type": "bad" },
+      { "u_add_effect": "bleed", "intensity": 1, "bodypart": "head" },
+      { "u_set_hp": { "math": { "u_hp('head') - 1" } }, "target_part": "head" }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_STAMINA_LOSS",
+    "condition": { 
+      "and": [ 
+        { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", ">=", "60" ] },  
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "( (u_val('vitamin', 'name:vitamin_psionic_drain') * 2.2 ) + 30)"
+              ]
+            },
+            "y": 1000
+          }
+        } 
+      ] 
+    },
+    "effect": [
+      { "u_message": "As you unleash your powers, your vigor leaves you in a rush!", "type": "bad" },
+      { "math": [ "u_val('stamina')", "-=", "rng(3000,9000)" ] }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_DRAIN_EFFECT_CHECK_FATIGUE",
+    "condition": { 
+      "and": [ 
+        { "math": [ "u_val('vitamin', 'name:vitamin_psionic_drain')", ">=", "60" ] },  
+        {
+          "x_in_y_chance": {
+            "x": {
+              "math": [
+                "( (u_val('vitamin', 'name:vitamin_psionic_drain') * 1.2 ) + 80)"
+              ]
+            },
+            "y": 1000
+          }
+        } 
+      ] 
+    },
+    "effect": [
+      { "u_message": "As you unleash your powers, you feel weary!", "type": "bad" },
+      { "math": [ "u_val('fatigue')", "-=", "rng(30,90)" ] }
```

```suggestion
      { "math": [ "u_val('fatigue')", "+=", "rng(30,90)" ] }
```

I think you meant to increase fatigue, not decrease it.